src/EventGrid/EventGrid/EventSubscription/NewAzureEventGridSubscription.cs

```diff
@@ -194,7 +335,16 @@ public class NewAzureEventGridSubscription : AzureEventGridCmdletBase
         [Parameter(
             Mandatory = false,
             ValueFromPipelineByPropertyName = true,
-            Position = 5,
```

Comment:
@ahamad-MS make sure that the `Position` remains the same in the `ResourceGroupNameParameterSet` set

Reply:
Fixed. This was missed somehow. thanks for the catch.